src/item_factory.cpp

```diff
@@ -1422,6 +1444,8 @@ void Item_factory::load_basic_info(JsonObject &jo, itype *new_item_template)
         new_item_template->category = get_category( jo.get_string( "category" ) );
     }
 
+    assign( jo, "faults", new_item_template->faults );
```

Comment:
I'm not sure that applies so much here - where is the extra implementation (or conversely how is the implementation simplified by pushing `faults` to the `engine` slot)?